app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -279,6 +283,8 @@ def parse_instance(instance)
           :orchestration_stack => @data_index.fetch_path(:orchestration_stacks, @resource_to_stack[uid]),
           :availability_zone   => @data_index.fetch_path(:availability_zones, 'default'),
           :resource_group      => @data_index.fetch_path(:resource_groups, rg_ems_ref),
+          :labels              => labels,
+          :tags                => map_labels('Vm', labels),
```

Comment:
FYI I think this should be 'VmAzure' like in #231, this is based on top of older version of BZ-1493041-old-refresh branch...

Reply:
I'm going to rebase and add some changes to this PR after https://github.com/ManageIQ/manageiq-providers-azure/pull/231 will be merged